packages/transformers/webextension/src/schema.js

```diff
@@ -477,6 +477,12 @@ export const MV3Schema = ({
       },
       additionalProperties: false,
     },
+    side_panel: {
+      type: 'object',
+      properties: {
+        default_path: string,
+      },
+    },
```

Comment:
Let's add this for consistency with the other schema items

```suggestion
      additionalProperties: false,
    },
```